### ISSUE TITLE
Maven support ?

### ISSUE THREAD

**Issue by fbacchella** [HaraldKi/monqjfa#3]
Do you plan to publish this library on maven, or on jfrog ? I will then be able to include it in my regex benches: https://github.com/fbacchella/RegexPerf.

**Comment by HaraldKi** [comment]
Good hint, I should finally get around and do this. But its not really a plan with a deadline yet.-) Does your benchmark take care of different regex syntaxes? Because mine is quite non-standard in some regards.

**Comment by fbacchella** [comment]
No, it's purpose is to test different implementations of java.util.regex. The tested regex are here:     https://github.com/fbacchella/RegexPerf/blob/7d56c996d17da3305ef3da44e87ebc9b5d6e0f55/src/main/java/loghub/Runner.java#L32. But I could (should?) split the test in basic vs perl/advanced regex.
